services/frontend-v3/src/components/profileForms/welcome/WelcomeView.jsx

```diff
@@ -68,6 +77,10 @@ const WelcomeView = ({ gedsProfiles, intl, load, userId, history }) => {
       fontStyle: "italic",
       marginTop: "-4px",
     },
+    skipButton: {
+      marginTop: 20,
```

Comment:
I know you didn't write this but do you think you could make a WelcomeView.scss file and move these there?  So that way we are making a slow migration